tests/backends/tests.py

```diff
@@ -970,9 +970,7 @@ def runner():
             t.start()
             t.join()
         # Each created connection got different inner connection.
-        self.assertEqual(
-            len(set(conn.connection for conn in connections_dict.values())),
-            3)
+        self.assertEqual(len({conn.connection for conn in connections_dict.values()}), 3)
```

Comment:
This line is 89 characters long. I was under the impression the Django code doesn't push hard for an 80 character length limit, but instead an 100-120 character limit. This isn't even the longest line in this same file.

From https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/

> An exception to PEP 8 is our rules on line lengths. Don’t limit lines of code to 79 characters if it means the code looks significantly uglier or is harder to read. We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult. This check is included when you run flake8. Documentation, comments, and docstrings should be wrapped at 79 characters, even though PEP 8 suggests 72.